src/test/scala/edu/ie3/mobsim/MobilitySimulatorSpec.scala

```diff
@@ -290,5 +306,48 @@ class MobilitySimulatorSpec extends UnitSpec with MobilitySimulatorTestData {
         actualNextEvent shouldBe expectedNextEvent
       }
     }
+
+    "update electricVehicles correctly" in {
+      val updateElectricVehicles =
+        PrivateMethod[Unit](Symbol("updateElectricVehicles"))
+      val mobSimCopy = mobSim
```

Comment:
This is _not_ how you make a copy. Here, you're just creating another reference to the same object.

Reply:
Since testing it hasshown that a copy is not needed, I will remove this line and adapt the test.